### PR TITLE
Stop hanging forever for non-terminated thread in test

### PR DESCRIPTION
**Pull request by yjshen**

### CODE REVIEW

File: pulsar-io/flume/src/test/java/org/apache/pulsar/io/flume/node/TestApplication.java [hunk 1]
```diff
@@ -117,7 +117,7 @@ public void testBasicConfiguration() throws Exception {
         eventBus.post(materializedConfiguration);
         application.start();
 
-        Thread.sleep(1000L);
+        Thread.sleep(1003L);
```
Comment (codelipenghui) [inline pulsar-io/flume/src/test/java/org/apache/pulsar/io/flume/node/TestApplication.java]: Is this a meaningful change?

Comment (yjshen) [reply pulsar-io/flume/src/test/java/org/apache/pulsar/io/flume/node/TestApplication.java]: no, just to trigger tests. Otherwise, GitHub action won't run.

**Pull request closed without merging**